api/resolvers/video-resolver.ts

```diff
@@ -13,9 +13,7 @@ export class VideoResolver {
     @Mutation(() => Video)
     protected async uploadVideo(
         @Arg('video', () => GraphQLUpload)
-        { createReadStream, filename }: Upload,
-        @Arg('mimeType', () => String)
-        mimeType: string,
+        { createReadStream, filename, mimetype }: Upload,
```

Comment:
I would do a check to make sure that the mimetype is a video, check for the video files output from ios and android specifically as well as the common mp4, mov, etc